rbi/bios-256k/readme.md

```diff
@@ -0,0 +1,24 @@
+# RB for bios-256k.bin
+
+## files
```

Comment:
Please add am empty line between the chapter title and content.

Reply:
thanks, have added